.github/workflows/build-test-ubuntu20.yml

```diff
@@ -44,6 +44,31 @@ jobs:
           # not realy needed
           CMAKE_C_COMPILER: ${{ matrix.c-compiler }}
 
+      - name: Collect Timings
+        run: |
+          inFolder=./build/${{matrix.config}}/
+          outFolder=./build/${{matrix.config}}/${{ fromJSON('["Clang_11", "GCC_10"]')[matrix.compiler == 'GCC 10'] }}/Timings
+          mkdir -p $outFolder
+          folderList=(MRCommonPlugins mrviewerpy MRTest MRViewerApp MRViewer mrmeshpy meshconv mrmeshnumpy MRMesh)
+          for folder in ${folderList[*]}
```

Comment:
I think, we can merge all log files:
```
rm -rf ./$folderName || true
for file in `find ./build/${{matrix.config}}/*/  -name compile_timings.txt`
do
  cat $file >> ./$folderName/compile_time.txt
done
```
same for `link_timings.txt`
then just zip/tar `$folderName` and upload

Reply:
reworked